map/style_tests/style_symbols_consistency_test.cpp

```diff
@@ -56,11 +54,43 @@ set<string> GetSymbolsSetFromResourcesFile(string const & density)
   set<string> symbols;
   SdfParsingDispatcher dispatcher(symbols);
   ReaderPtr<Reader> reader = GetStyleReader().GetResourceReader("symbols.sdf", density);
-  ReaderSource<ReaderPtr<Reader> > source(reader);
+  ReaderSource<ReaderPtr<Reader>> source(reader);
   ParseXML(source, dispatcher);
   return symbols;
 }
 
+using SymbolsMapping = map<string, map<int, string>>;  // ClassifName -> Scale, SymbolName
+
+SymbolsMapping GetCurrentStyleSymbolsMapping()
+{
+  SymbolsMapping mapping;
+
+  string drawingRules;
+  GetStyleReader().GetDrawingRulesReader().ReadAsString(drawingRules);
+
+  ContainerProto cont;
+  cont.ParseFromString(drawingRules);
+  for (int ci = 0; ci < cont.cont_size(); ++ci)
+  {
+    ClassifElementProto const & classifElement = cont.cont(ci);
+    for (int ei = 0; ei < classifElement.element_size(); ++ei)
+    {
+      DrawElementProto const & drawElement = classifElement.element(ei);
+      if (drawElement.has_symbol())
+      {
+        mapping[classifElement.name()][drawElement.scale()] = drawElement.symbol().name();
```

Comment:
@kshalnev, could you please tell me, do we need to check that there are no symbols with duplicating names and scales?